modules/tooling-support/src/main/java/org/mule/runtime/module/tooling/internal/utils/ArtifactHelper.java

```diff
@@ -108,7 +110,16 @@ public void visit(ConfigurationElementDeclaration declaration) {
   }
 
   public Optional<ConfigurationInstance> getConfigurationInstance(String configName) {
-    return findConfigurationProvider(configName).map(cp -> cp.get(getNullEvent()));
+    return findConfigurationProvider(configName).map(cp -> {
```

Comment:
Why is this?

Reply:
correct, creating an event is not something to be done lightly and it must be properly disposed/finished afterwards.